requirements/common.txt

```diff
@@ -5,7 +5,6 @@ django-import-export==0.5.1
 djangorestframework==3.5.2
 djangorestframework-expiring-authtoken==0.1.4
 django-cors-headers==1.3.1
-django-rest-auth[with_social]==0.9.2
```

Comment:
@party98 Can you please tell me why you have removed this file?

Reply:
This package handles password authentication. During this process, it strips the password before processing, thus removing spaces at the beginning and end of the password input and thereby mutating the password. As you should be able to see, I have replaced this with a fork of the same package which does not remove the spaces. 
I have also made a PR for the given package containing the same fix.

Reply:
@party98 So, how does this explains that Travis fail is due to it?

Reply:
Travis should not be failing due to this, as the replaced package is the fork of the same package with just one line changed

Reply:
I'll try to look into the cause of the failure.